applications/asset_tracker_v2/README.rst

```diff
@@ -245,18 +267,32 @@ Check and configure the following configuration options for the application:
 
    This application configuration sets a custom client ID for the respective cloud. For setting a custom client ID, you need to set :option:`CONFIG_CLOUD_CLIENT_ID_USE_CUSTOM` to ``y``.
 
-Additional configuration
-========================
 
-To get the application to communicate with a specified cloud service, configure the Kconfig options specific to each *cloud library*.
-Every cloud service supported in |NCS| has a corresponding *cloud library* that needs to be selected and properly configured.
+.. _mandatory_config:
+
+Mandatory library configuration
+===============================
+
+You can set the mandatory library-specific Kconfig options in the :file:`prj.conf` file of the application.
 
-You must check and configure the following :ref:`lib_aws_iot` library options that are used by the application:
+Configurations for AWS IoT library
+----------------------------------
 
 * :option:`CONFIG_AWS_IOT_BROKER_HOST_NAME`
 * :option:`CONFIG_AWS_IOT_SEC_TAG`
 
-Additionally, you can add the following optional configurations to configure the heap or to provide additional information such as APN to the modem for registering with an LTE network:
+
+Configurations for Azure Iot Hub library
+----------------------------------------
+
+* :option:`CONFIG_AZURE_IOT_HUB_DPS_ID_SCOPE`
```

Comment:
```suggestion
* :option:`CONFIG_AZURE_IOT_HUB_DPS_HOSTNAME`
* :option:`CONFIG_AZURE_IOT_HUB_DPS_ID_SCOPE`
```

Reply:
Just to clarify, referring to the comment made by @jtguggedal in https://github.com/nrfconnect/sdk-nrf/pull/3950#discussion_r636767769. Why would we need to set the DPS hostname if the global default alternative can be used to provision the device? `CONFIG_AZURE_IOT_HUB_DPS_HOSTNAME` defaults to `global.azure-devices-provisioning.net`

Reply:
Right, good point. I've reviewed the docs, and it can be used if customers don't use a private network, however, [if they do, they need to change it](https://docs.microsoft.com/en-us/azure/iot-dps/virtual-network-support). So the default value `global.azure-devices-provisioning.net` will not always work and should be reviewed by the user.